tests/foreman/ui/test_organization.py

```diff
@@ -301,7 +299,7 @@ def test_positive_download_debug_cert_after_refresh(session):
     """
     org = entities.Organization().create()
     try:
-        upload_manifest_locked(org.id, original_manifest())
+        target_sat.upload_manifest(org.id)
```

Comment:
Same issue as above, this will need either a manifest fixture or `function_entitlement_manifest_organization`.

Reply:
I made the changes in upload_manifest method itself and it's working. Will that be good?

Reply:
Yes, that's fine for now.